### PR TITLE
test: coverage for tools dir

### PR DESCRIPTION
**Pull request by nkrantz**
# In this PR

[DSYS-3383](https://issues.corp.twilio.com/browse/DSYS-3383)

Adds unit tests to for the following files:

```
paste-icons/
├── tools/
│   ├── actions/
│   │   ├── convertAllAction.ts
│   │   ├── convertNewAction.ts
│   │   ├── listIconsAction.ts
│   ├── templates/
│   │   ├── buildListTemplate.ts
│   │   ├── jsonTemplate.ts
│   │   ├── reactIconTemplate.ts
│   ├── utils.ts/
```

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: packages/paste-icons/__test__/icons.spec.tsx [hunk 1]
```diff
@@ -46,3 +62,91 @@ describe('Icons', () => {
...
+    });
+  });
+
+  describe('tools/utils.ts utils', () => {
+    it('getOutputComponentName util returns the expected output', () => {
+      expect(getOutputComponentName('brandLogoIcon.svg')).toEqual('BrandLogoIcon');
+    });
+
+    it('pascalCaseWordSplitter util returns the expected output', () => {
+      expect(pascalCaseWordSplitter('hElLo iM nORA')).toEqual('h El Lo i M n ORA');
```
Comment (gloriliale) [inline packages/paste-icons/__test__/icons.spec.tsx:121]: 😂 beautiful.

File: packages/paste-icons/__test__/icons.spec.tsx [hunk 2]
```diff
@@ -46,3 +62,91 @@ describe('Icons', () => {
...
+    it('getReactOutputPath util returns the expected output', () => {
+      expect(getReactOutputPath(fileName).includes(`paste/packages/paste-icons/src/IAmAFileTsxIcon.tsx`)).toBeTruthy();
+    });
+    it('getBuildFileName util returns the expected output', () => {
+      expect(getBuildFileName(removeTsxExtension(fileName))).toEqual(`src/${fileName}`);
+    });
+
+    it('maybeHandleError util throws if error is present', () => {
+      // hide console errors from terminal when throwing expected errors
+      const spy = jest.spyOn(console, 'error');
```
Comment (gloriliale) [inline packages/paste-icons/__test__/icons.spec.tsx:145]: praise: I had not known about this functionality, very cool!

**Review by gloriliale — APPROVED** [review]
This looks great! So excited that there's so many tests now!

File: packages/paste-icons/__test__/icons.spec.tsx [hunk 3]
```diff
@@ -46,3 +62,91 @@ describe('Icons', () => {
     });
   });
 });
+
+describe('icons util functions', () => {
+  const testIconList = ['icon1', 'icon2', 'icon3'];
+  const fileName = 'iAmAFile.tsx';
+
+  describe('tools/templates/buildListTemplate util', () => {
+    it('returns the expected output', () => {
```
[the remark this answers is not in the document]
Comment (nkrantz) [reply packages/paste-icons/__test__/icons.spec.tsx:71]: totally, I actually meant to clean this stuff up and forgot. will push up changes shortly

**Review by zahnster — APPROVED** [review]

**Review by TheSisb — APPROVED** [review]
Very nice!

**Pull request merged**